runtime/vanilla/src/lib.rs

```diff
@@ -220,14 +225,16 @@ impl pallet_balances::Config for Runtime {
 }
 
 parameter_types! {
-    pub const TransactionByteFee: Balance = 1;
+    // 1/10 of Kusama's transaction fee
+    pub const TransactionByteFee: Balance = 1 * MILLICENTS;
 }
 
 impl pallet_transaction_payment::Config for Runtime {
+    // TODO add missing DealWithFees
```

Comment:
Is this still needed?

Reply:
I'd like to add it, because statemint will give fees to collators so that they have some interests to run collators, but now we don't have

Reply:
so we need to finish collator selection first and then add `DealWithFees` so that collator can receive part of fees as incentives